lib/pages/athlete/AthletePage.dart

```diff
@@ -987,29 +987,34 @@ class _AthletePageState extends State<AthletePage> {
                       ),
                     ],
                   ),
-                  SizedBox(width: 30),
-                  Container(
-                    width: 100,
+                  //SizedBox(width: 0),
```

Comment:
If you are going to comment something out and not tag a TODO then remove it

Reply:
💯